plugins/org.locationtech.udig.tool.select/src/org/locationtech/udig/tool/select/internal/AOILayerSelectionTool.java

```diff
@@ -235,16 +244,16 @@ public void run() {
         };
 
         @Override
-        protected IPreferenceStore fillFields( Composite parent ) {
+        protected IPreferenceStore fillFields(Composite parent) {
 
             Button nav = new Button(parent, SWT.CHECK);
-            nav.setText("Navigate");
+            nav.setText("Navigate"); //$NON-NLS-1$
```

Comment:
this is an UI element and should be exported to allow translations, please remove `//$NON-NLS-1$` here ;)